Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.0.xml

```diff
@@ -1,391 +1,461 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <ns4:modelInfo name="QDM" url="urn:healthit-gov:qdm:v5_0_draft" targetQualifier="qdm" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ns4="urn:hl7-org:elm-modelinfo:r1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-               patientClassName="QDM.Patient" patientClassIdentifier="Patient" patientBirthDatePropertyName="birth datetime">
+               patientClassName="QDM.Patient" patientClassIdentifier="Patient" patientBirthDatePropertyName="birthDatetime" version="5.0">
     <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.Patient" baseType="System.Any" identifier="Patient">
-        <ns4:element name="birth datetime" type="System.DateTime"/>
+        <ns4:element name="birthDatetime" type="System.DateTime"/>
     </ns4:typeInfo>
     <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.PhysicalExamOrder" retrievable="false" baseType="QDM.QDMBaseType">
-        <ns4:element name="authorTime" type="DateTime"/>
-        <ns4:element name="reason" type="Code"/>
-        <ns4:element name="method" type="Code"/>
-        <ns4:element name="anatomicalLocationSite" type="Code"/>
-        <ns4:element name="negationRationale" type="Code"/>
+        <ns4:element name="authorTime" type="System.DateTime"/>
+        <ns4:element name="reason" type="System.Code"/>
+        <ns4:element name="method" type="System.Code"/>
+        <ns4:element name="anatomicalLocationSite" type="System.Code"/>
+        <ns4:element name="negationRationale" type="System.Code"/>
     </ns4:typeInfo>
     <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.PatientCharacteristicSex" retrievable="false" baseType="QDM.QDMBaseType"/>
     <ns4:typeInfo xsi:type="ns4:ClassInfo" name="QDM.EncounterActive" retrievable="false" baseType="QDM.QDMBaseType">
-        <ns4:element name="relevantPeriod" type="DateTimeInterval"/>
-        <ns4:element name="reason" type="Code"/>
-        <ns4:element name="facilityLocation" type="Code"/>
-        <ns4:element name="locationPeriod" type="DateTimeInterval"/>
-        <ns4:element name="lengthOfStay" type="Quantity"/>
+        <ns4:element name="relevantPeriod" type="interval&lt;System.DateTime&gt;"/>
```

Comment:
is there a different approach that could be used to signify an interval? The escaped characters make it hard to read in this format.